autometacal/python/metacal.py

```diff
@@ -81,13 +81,26 @@ def generate_mcal_image(gal_images,
 
   # Compute inverse Fourier transform
   img = tf.math.real(tf.signal.fftshift(im_reconv))
+  
+  img=img[:,fact*nx:-fact*nx,fact*ny:-fact*ny]
+
+  return img
+
+def generate_fixnoise(shape,psf_image,reconvolution_psf_image,g):
+  noise = tf.random.normal(shape)
+  rotnoise = tf.image.rot90(noise)
```

Comment:
that's not necessary here to do this rotation. Would only be needed I guess if your noise image was not just a simple realisation from a standard guassian. ie.e had non-stationary variance and/or some directional correlations

Reply:
Yeah you right. I told André to do it based on what they do in ngmix. But they handle the case of non-stationary noise. Didn't thought about that. That's on me.

Reply:
So, since from the argument below, we'll be keeping the noise realisation outside the fixnoise generator, I should keep the initial rotation, shouldn't it? Because then I can input non-stationary noise without problems.
The function will look something like:

```
def generate_fixnoise(noise,psf,psf_reconv,g):
  rot_noise = tf.image.rot90(noise)
  mcal_noise = amc.generate_mcal_image(rotnoise, psf, reconv_psf, g)
  final_noise =  tf.image.rot90(mcal_noise,k=3)
  return final_noise
```

Reply:
That looks good to me. You won't have to change this later.   
The only drawback I see is that it could overload the traceback of the gradient for basic tests but a simple rotation is probably not too bad.